.github/workflows/hardware-smoke.yml

```
@@ -236,3 +236,23 @@ jobs:
           name: E2E Flash results (${{ matrix.config.board }})
           path: |
             zephyr/twister-e2e-flash/**/update.fwbundle
+
+      - name: run-e2e-reset-test
+        working-directory: zephyr
+        run: |
+          # Run a full flash test, using tt-flash as the runner
+          ./scripts/twister -i -p $SMC_BOARD \
+            --tag e2e-reset -T ../tt-zephyr-platforms/app \
+            --west-flash="--force" \
+            --west-runner tt_flash \
+            --device-testing -c -ll DEBUG \
+            --device-serial-pty rtt \
+            --outdir twister-e2e-reset
+
+      - name: Upload E2E Reset Results
+        if: ${{ always() }}
+        uses: actions/upload-artifact@v4
+        with:
+          name: E2E Reset results (${{ matrix.config.board }})
+          path: |
+            zephyr/twister-e2e-reset/**/update.fwbundle
```

I think this can be removed. We probably don't need to upload more firmware bundles.

```suggestion
      - name: Upload E2E Reset Results
        if: ${{ always() }}
        uses: actions/upload-artifact@v4
        with:
          name: E2E Reset results (${{ matrix.config.board }})
          path: |
            zephyr/twister-e2e-reset/**/update.fwbundle
```